README.md

```diff
@@ -357,7 +386,7 @@ This can be useful alongside `event='none'` when you only want to toolip to show
 
 ```hbs
 {{!--Binds the tooltip visibility to the showTooltip property--}}
-{{tooltip-on-component isShown=showTooltip}}
```

Comment:
why did you delete this? Also, the Controller has a `showTooltips` property. I think this is fine to leave but the Controller property should be updated to `showTooltip`.